.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "markdown_metayaml"]
```

Comment:
Might have been some time ago. I found `authors` (similar `tags`), specified one per line as in the [Markdown Meta extension example](https://pythonhosted.org/Markdown/extensions/meta_data.html) and hence parsed as lists of strings instead of single strings of comma-joined values, working just as well.